documentation/book/proc-upgrading-the-cluster-operator.adoc

```diff
@@ -0,0 +1,50 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrade.adoc
+
+[id='proc-upgrading-the-cluster-operator-{context}']
+= Upgrading the Cluster Operator from 0.9.0 to 0.10.0
```

Comment:
I think this whole section is dangerously specific:
* It doesn't apply to the downstream versions
* It will be completely lost in the documentation
* We would need to update this for every single release

I understand that we might need this as a specific guide with specific steps and numbers, but shouldn't this be in the release notes instead?

Reply:
I'm fine with this content being in the release notes instead, but we don't have release notes in the repo for me to put this content there.

Reply:
Well, we should probably start with it somewhere. Maybe we can add that as some first section of the documentation? I think the Red Hat docs were sometimes doing it that way.

While the issue about updating this every release can be solved also later, we need to make this more prominent to make sure people don't come and just do `kubectl apply -f ...`. Not sure whether linking it from the GitHub release page is sufficient.

@laidan6000 @ncbaratta: Could we get any advice here? As the release notes with some upgrade guide etc will be needed also for downstream ... is there any preference for how to display them / format them / present them etc.?